src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/MessageReader.java

```diff
@@ -30,122 +36,58 @@
 public class MessageReader implements ItemReader<PaymentTransactionCreatedOrUpdatedMessage> {
 
     private static final Logger LOG = LoggerFactory.getLogger(MessageReader.class);
-
+    private static final String PAYMENT_TRANSACTION_STATE_CHANGED = "PaymentTransactionStateChanged";
+    private static final String PAYMENT_TRANSACTION_ADDED = "PaymentTransactionAdded";
+    final int RESULTS_PER_PAGE = 500;
```

Comment:
Please check and remove ALL redundant fields.

Reply:
7bf82ce